packages/smooth_app/lib/generic_lib/theme_constants.dart

```diff
@@ -0,0 +1,3 @@
+const String THEME_SYSTEM_DEFAULT = 'System Default';
```

Comment:
the strings need to be made translatable

Reply:
No, these aren't shown anywhere but are for storing the values in the preferences.

Reply:
But what about putting them into theme_provider.dart a new file for those three lines seems a bit over the top doesn't it